service/frontend/clusterRedirectionPolicy.go

```diff
@@ -83,6 +109,28 @@ var selectedAPIsForwardingRedirectionPolicyAPIAllowlist = map[string]struct{}{
 	"ResetWorkflowExecution":           {},
 }
 
+// selectedAPIsForwardingRedirectionPolicyAPIAllowlistV2 contains a list of non-worker APIs which can be redirected.
+// This is paired with DCRedirectionPolicySelectedAPIsForwardingV2 - keep both lists up to date.
```

Comment:
do we have a unit test for this maybe? (assert.Equal(t, len(selectedAPIsForwardingRedirectionPolicyAPIAllowlistV2), len(DCRedirectionPolicySelectedAPIsForwardingV2))

Reply:
yea, probably a good idea.  lemme write one.  I can also ensure it's a superset with one.